### DISCUSSION TITLE
config.ini should let users choose the navigation tile for plugins.

### DISCUSSION THREAD

**Discussion by joeblubaugh** [grafana/grafana#62546] [Plugins]
The new navigation experience has a configurable set of "sections" on the side. The config setup allows users to choose the section they'd like a plugin's UI to appear in. This is used in the Grafana codebase to put the Oncall plugin inside the "Alerting and Incidents" section:
```go
https://github.com/grafana/grafana/blob/e3b4da2279100ce0f2592175b588b443fb1faa82/pkg/services/navtree/navtreeimpl/applinks.go#L260
```

In that code, plugins can also be configured with a custom title for the nav section. This custom title cannot be configured in config.ini. 

I'd like for users to be able to set a custom title in the config file just as happens in the code. For example, you might have a plugin with a name like "Google Cloud Stackdriver" that provides some UI, but in the nav some users might like to just call it "Stackdriver" to save space.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82866. Feel free to continue the discussion around this there. Thank you!
